sdk/build.gradle

```diff
@@ -0,0 +1,48 @@
+subprojects {
```

Comment:
This is the SDK build script. It makes a `subprojects` call so that all the contained build logic is applied to all its subprojects. Any build logic common to all SDK modules should be declared here.
Nothing is applied on this project level since this level is not designed to contain anything but the individual SDK modules that it builds out